app/controllers/physical_ticket_controller.rb

```diff
@@ -13,6 +13,8 @@ def show
     @file_name = "ticket_for_#{@conference.short_title}"
     @user = @physical_ticket.user
     @ticket_layout = @conference.ticket_layout.to_sym
+    @qrcode = RQRCode::QRCode.new(@physical_ticket.token)
+    @png = @qrcode.as_png(size: 180, border_modules: 0)
```

Comment:
Use a better name for this variable, I mean something like `@qrcode_image` (we shouldn't care about the png format in the view, is just an image of the QRCode)